### ISSUE TITLE
All DM are rooms with update to 1.4.4

### ISSUE THREAD

**Issue by 532910** [vector-im/element-android#5526]
1.4.4 [40104042] (G-b6763) and a build before from GPlay beta

HS synapse 1.54.0+bullseye1 from packages.matrix.org

![Screenshot_20220312-035147_Element.png](https://user-images.githubusercontent.com/14617699/157996543-c5e4bd9e-4631-4d26-b41e-ed8fcf2bc5c8.png)

**Comment by ouchadam** [comment]
thanks for raising, I haven't been able to reproduce locally, are there any other steps to be aware of? will continue to investigate on my side

**Comment by 532910** [comment]
I dunno. The same version in [shelter](https://github.com/PeterCxy/Shelter) works fine with matrix.org

**Comment by ouchadam** [comment]
have the DMs ended up somewhere else in the app, like the rooms tab or within a space, or have they completely disappeared?

**Comment by 532910** [comment]
Yep, all DMs are moved to the rooms tab. I tried to say this in subject.

**Comment by ouchadam** [comment]
I'm still unable to reproduce this issue and no other reports have come in so far, going to downgrade from critical 

- does enabling and disabling `Settings -> Preferences -> Show all rooms in Home` make any difference?
- does interacting with the DMs sending/receiving messages cause them to shift back to being DMs?
- as a last resort, does a `Settings -> General -> Clear cache` solve the issue?

**Comment by 532910** [comment]
> * as a last resort, does a `Settings -> General -> Clear cache` solve the issue?

Tried before reporting this issue.

> * does interacting with the DMs sending/receiving messages cause them to shift back to being DMs?

Nope.

> * does enabling and disabling `Settings -> Preferences -> Show all rooms in Home` make any difference?

This works as it should: hides DMs belonging to some space from Home. In both states of this switch DM tab is empty (as it shown on the screenshot) for all spaces.

**Comment by 532910** [comment]
Just found a workaround!!
I typed `/converttoroom` in one of my favorite DMs in the desktop client and immediately after that all DMs moved back to DM tab in android. I switched that room back with `/converttodm` and now android looks as before upgrade.

**Comment by ouchadam** [comment]
that's great to hear! I'll raise with the other teams as this sounds like there may have been a state reset (causing the direct message information to be lost)

**Comment by mcg-matrix** [comment]
Happened to me when upgrading Element-FDroid from 1.4.8 to 1.4.11. No other sessions showed the same effect (Element-Desktop, several browser sessions, an old Riot-FDroid installation, ...). I've got only that single Element-FDroid session for that account, though, so can't say whether another Element-FDroid session would have shown the same effect.

> I typed `/converttoroom` in one of my favorite DMs in the desktop client and immediately after that all DMs moved back to DM tab in android. I switched that room back with `/converttodm` and now android looks as before upgrade.

Thanks for that hint! Issuing `/converttodm` in one of the affected rooms using Element-Desktop (Linux) immediately moved all the DM-rooms back to the "Direct Messages" tab in Element-Android. I issued just that single `/converttodm`, no other `/convertto*` in any other room.
